docs/config-files/config.py

```diff
@@ -0,0 +1,25 @@
+# Reference:
+
+# brew install libyaml
+# sudo python -m easy_install pyyaml
+# http://net-informations.com/python/file/open.htm
+# https://www.basinc.com/manuals/EC_epsilon/Techniques/ChronoI/ca
+
+import yaml
+
+def main():
+    config_file = open("config.yml","r") # open a file
+
+    loaded = yaml.load(config_file)
```

Comment:
These days it is recommended to use ``yaml.safe_load`` (and ``safe_dump``).  It won't allow malicious code to be run without your knowledge.  Sorry, I didn't use that in my example.